org.civicrm.reqangular/src/common/services/session.js

```diff
@@ -42,4 +42,4 @@
       };
     }]);
   });
-})(CRM, Drupal);
+})(CRM, window.Drupal);
```

Comment:
Okay then I think it is fine. But we should use camel casing for anything related to javascript i.e. `contactID` or `contactId`